packages/app/non-translatable-markup/vite.config.ts

```diff
@@ -11,6 +11,7 @@ export default defineConfig({
   dependencies: Object.keys(packageJson.peerDependencies),
   test: {
     coverage: {
+      exclude: ['src/**/index.ts'],
```

Comment:
do we need all the internal barrel files?

Reply:
Going to merge as I would need these changes to continue with my task but if you have concerns, please feel free to leave them here and I will address them on follow-up

Reply:
there are two index.ts files - the root one and one inside `methods`. Is second one useful?

Reply:
Oh, I see your point, and at the beginning, I thought is was going to be bigger, but looking at the result ... it doesn't make too much sense ... will remove it on follow-up PR